README.md

```diff
@@ -403,7 +414,7 @@ def generate_launch_description():
     ])
 ```
 
-## Using with composition
+### Using recorder and player as composable nodes
```

Comment:
The link to this section under _Using rosbag2_ needs to be updated. Search for `#using-with-composition`.